src/messages.js

```diff
@@ -219,7 +220,8 @@ var warnings = {
   W135: "{a} may not be supported by non-browser environments.",
   W136: "'{a}' must be in function scope.",
   W137: "Empty destructuring.",
-  W138: "Regular parameters should not come after default parameters."
+  W138: "Regular parameters should not come after default parameters.",
+  W139: "Function declarations should not be used after the `instanceof` operator."
```

Comment:
It's actually, "Function expressions" that we are warning about. Also, let's
say "as the second operand to instanceof".